server/middleware/forceHTTPS.js

```diff
@@ -0,0 +1,12 @@
+function forceHTTPS(req, res, next) {
+
```

Comment:
nit: Let's remove the whitespace inside this function (the extra newlines), and throughout indentation can be just 2 spaces